### DISCUSSION TITLE
Can you disable the "You can't access containers during pvp"?

### DISCUSSION THREAD

**Discussion by ArvickC** [GriefPrevention/GriefPrevention#1094] [Q&A]
The answer to your question may exist in the wiki, go check that out first! 

Wiki: https://github.com/TechFortress/GriefPrevention/wiki

Otherwise, if you have a spigot account, you may have better luck posting in the discussion thread so people who follow the thread may see your question. https://www.spigotmc.org/threads/griefprevention.35615/page-311

Still no luck? Really? Then go ahead and ask here.

Please remember to provide as much detail as you can - smart questions are
much more likely to be answered.  See "Asking questions the smart way":
http://www.catb.org/~esr/faqs/smart-questions.html

As an open-source project, we're all volunteers with limited time to spend,
so providing enough info (including any error messages etc) up-front makes
it more likely we can provide you an answer with limited time.

**Hello! I was just wondering if there is a way where I can make it so you can open containers whilst in pvp?**

**Comment by RoboMWM** [comment]
No - would recommend just disabling the timer

Eventually this will be moved out into an addon, since I too would like to configure how this feature works.
